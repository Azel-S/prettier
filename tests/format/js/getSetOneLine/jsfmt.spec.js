run_spec(__dirname, ["babel", "flow", "javascript"], {getSetOneLine: false});
run_spec(__dirname, ["babel", "flow", "javascript"], {getSetOneLine: true});