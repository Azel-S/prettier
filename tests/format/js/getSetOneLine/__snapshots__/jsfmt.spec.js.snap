// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getSetOneLine.js - {"getSetOneLine":false} format 1`] = `
====================================options=====================================
getSetOneLine: false
parsers: ["babel", "flow", "javascript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const objGet = {
    get name() {
      return this.name;
    },
  };

const objGetTwo = {
    get nameTwo() {
        this.name = this.name + " is valid";
        return this.name();
    },
}

const objSet = {
    set name(name) {
      return this.name = name;
    },
};

  const objSetTwo = {
    get name() {
      return this.name + "may not be valid in this context";
    },
};
  

=====================================output=====================================
const objGet = {
  get name() {
    return this.name;
  },
};

const objGetTwo = {
  get nameTwo() {
    this.name = this.name + " is valid";
    return this.name();
  },
};

const objSet = {
  set name(name) {
    return (this.name = name);
  },
};

const objSetTwo = {
  get name() {
    return this.name + "may not be valid in this context";
  },
};

================================================================================
`;

exports[`getSetOneLine.js - {"getSetOneLine":true} format 1`] = `
====================================options=====================================
getSetOneLine: true
parsers: ["babel", "flow", "javascript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
const objGet = {
    get name() {
      return this.name;
    },
  };

const objGetTwo = {
    get nameTwo() {
        this.name = this.name + " is valid";
        return this.name();
    },
}

const objSet = {
    set name(name) {
      return this.name = name;
    },
};

  const objSetTwo = {
    get name() {
      return this.name + "may not be valid in this context";
    },
};
  

=====================================output=====================================
const objGet = {
  get name() { return this.name; },
};

const objGetTwo = {
  get nameTwo() {
    this.name = this.name + " is valid";
    return this.name();
  },
};

const objSet = {
  set name(name) { return (this.name = name); },
};

const objSetTwo = {
  get name() { return this.name + "may not be valid in this context"; },
};

================================================================================
`;
